docker-compose.yml

```diff
@@ -421,8 +421,9 @@ volumes:
     external: true
   sentry-symbolicator:
     external: true
+  # not being external will still persist data across restarts
+  # it won't persist if someone does a docker compose down -v
   sentry-vroom:
-    external: true
   # These store ephemeral data that needn't persist across restarts.
```

Comment:
We should remove this comment as you suggested earlier to not confuse future folks. Definitely not accurate. (I mean we don't _need_ this data to persist through restarts but they do regardless)
```suggestion
```

Reply:
+1

Reply:
I added to the comment instead, noting we don't need to persist this data but it's persisted anyway. I think removing it removes information about the kind of data stored.